.circleci/run.sh

```diff
@@ -5,7 +5,7 @@ set -uexo pipefail
 HOST_DMD_VER=2.095.0
 CURL_USER_AGENT="CirleCI $(curl --version | head -n 1)"
 DUB=${DUB:-dub}
-N=${N:-2}
+N=${N:-$(getconf _NPROCESSORS_ONLN)}
```

Comment:
Why not `nproc` ? While `getconf` is POSIX-compliant, `_NPROCESSORS_ONLN` is not specified and some BSD-based systems such as FreeBSD may not provide this. I think we should check for existence and fallback if necessary. For FreeBSD and similar, something like `sysctl hw.ncpu` is also a possibility.

Reply:
I wasn't sure if this only ran on Linux.

Reply:
Right, this is circleci specific, so doesn't really matter :)